cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1209,7 +1243,7 @@ def default_for(
 
         .. versionadded:: 8.0.0
     '''):
-        with Conf('<platform name>', desc='''
+        with Conf('<platform name>', desc=f'''
```

Comment:
```suggestion
        with Conf('<platform name>', desc=dedent('''
```

This is how it's done elsewhere: https://github.com/cylc/cylc-flow/blob/6eb8f61c5b9777cb21ac68eff13aa4f0722f691a/cylc/flow/cfgspec/workflow.py#L1516-L1519